message/models.py

```diff
@@ -0,0 +1,68 @@
+from django.db import models
+from django.utils import timezone
+from dogowner.models import DogOwner
+from daycare.models import DayCare
+
+
+class AuthorOptions(models.TextChoices):
+    DogOwner = 'O', 'DogOwner'
+    DayCare = 'D', 'DayCare'
+
+
+class Message(models.Model):
+    author = models.CharField(max_length=1,
+                              choices=AuthorOptions.choices)
+    dogowner_id = models.ForeignKey(DogOwner, on_delete=models.CASCADE)
+    daycare_id = models.ForeignKey(DayCare, on_delete=models.CASCADE)
+    text = models.TextField()
+    date = models.DateTimeField(default=timezone.now)
+    Message_Reply = models.ForeignKey('self', on_delete=models.CASCADE, null=True)
+
+    def __str__(self):
+
+        if(self.author == AuthorOptions.DayCare):
+            res = f"From: {self.daycare_id}"
+            res += f" To: {self.dogowner_id}"
+        else:
+            res = f"From: {self.dogowner_id}"
+            res += f" To: {self.daycare_id}"
+
+        res += f" text: {self.text}"
+        res += f" date: {self.date}"
+        return res
+
+    @staticmethod
+    def create(author, dogowner_id, daycare_id, text):
+        last = Message.get_last_message_between_dogowner_daycare(
+            dogowner_id, daycare_id)
+
+        if(last is not None):
+            last = last.last()
```

Comment:
What is happening here?

Reply:
Maybe I should change the function name to get all msg between dog owner and daycare

Reply:
Got you

Reply:
Fixed, moved this logic to get_last_message function as described above!